src/services/todo.services/createToDo.ts

```diff
@@ -1,5 +1,4 @@
 import { v4 as uuidv4 } from "uuid";
-import fs from "fs";
 
 type Body = { title: string; description: string };
```

Comment:
Name of the type is missleading. If it's an early representation of a Todo, then it should be called something like `NewTodoDto`